apps/projects/app/components/Content/Issues.js

```diff
@@ -404,6 +426,39 @@ class Issues extends React.PureComponent {
     })
   }
 
+  getRowRenderer = issuesSorted => ({
+    index,
+    key,
+    parent,
+    style
+  }) => {
+    const issueStyle = {}
+    if(index === 0) {
+      issueStyle.borderRadius = '3px 3px 0 0'
+    }
+    if(index === issuesSorted.length - 1) {
+      issueStyle.borderRadius = '0 0 3px 3px'
+    }
+    return (
+      <CellMeasurer
+        cache={cache}
+        columnIndex={0}
+        key={key}
+        parent={parent}
+        rowIndex={index}
+      >
+        <div style={style}>
+          <Issue
```

Comment:
We could probably remove this wrapping `div`, and pass the styles from `CellMeasurer` directly to `Issue`. That might make it more clear how to move those border-radius styles back to the CSS, too.

Reply:
I see what you're talking about!

Reply:
Nice. Removing that `<div>` solved the issue.